modules/snooker/src/main/ts/ephox/snooker/api/TableSize.ts

```diff
@@ -14,13 +14,13 @@ export interface TableSize {
   readonly getCellDelta: (delta: number) => number;
   readonly singleColumnWidth: (w: number, delta: number) => number[];
   readonly minCellWidth: () => number;
-  readonly setElementWidth: (cell: Element<HTMLTableCellElement>, amount: number) => void;
-  readonly setTableWidth: (table: Element<HTMLTableElement>, newWidths: number[], delta: number) => void;
+  readonly setElementWidth: (element: Element<HTMLElement>, width: number) => void;
+  readonly adjustTableWidth: (delta: number) => void;
```

Comment:
I imagine we'll do a major bump of Snooker with all these changes. That's not customer-facing though so doesn't affect the release

Reply:
Snooker is already tagged as being a major bump (see versions.txt)

Reply:
Turns out we hadn't but we should have when we deleted a heap of unused APIs previously. I've updated `versions.txt` to show a major bump instead of a minor now.